README.rst

```diff
@@ -52,8 +52,13 @@ about who's involved and how Horovod plays a role, read the LF AI `announcement
 
 .. contents::
 
+|
+
+Documentation
+-------------
 
-The full documentation and an API reference are published at https://horovod.readthedocs.io/en/latest.
+- `stable <https://horovod.readthedocs.io/en/stable>`_
+- `master <https://horovod.readthedocs.io/en/latest>`_
```

Comment:
Might want to call this "latest"

Reply:
Or just have the 1 link (not sure when someone would need to see the /stable version)

Reply:
In practice, I think we actually want to direct users to /stable first (stable is the latest released version, correct?).  In a lot of cases, users get confused because they `pip install horovod`, but the docs do not match with the version they have installed (because latest is always the unreleased master branch).

Ideally, I think it would be good if we could have specific versioned sets of documentation like /0.19.2 to make it more clear for users.  For now, stable seems to be the closest proxy to that, but the name isn't very informative.  Maybe "released" or "production".

What do you think?

Reply:
Ah, definitely want to point to the right version to reduce that confusion. Looking into how it works, I think /stable is the recommended version, as it's kept up to date with the highest release version (for tagged releases): https://docs.readthedocs.io/en/stable/versions.html

"Latest release" could work?

Reply:
Sounds good!  Changed to "Latest Release" instead.

Reply:
I agree, latest is a synonym for stable or latest release, master is a synonym for git head or latest source. Ideally, documentation of each released version should be available at horovod.readthedocs.io.